ambari-server/src/main/java/org/apache/ambari/server/bootstrap/BSRunner.java

```diff
@@ -282,17 +286,17 @@ public void run() {
 
       Process process = pb.start();
 
+      StringBuilder logInfoMessage = new StringBuilder(validationInstance ? "Validation" : "Bootstrap");
```

Comment:
Please extract `validationInstance ? "Validation" : "Bootstrap"` into a method, and reuse it below for `scriptLog`.